lib/config.js

```diff
@@ -340,6 +343,17 @@ config.start = function start() {
                          .pipe(errorLogger || logStream, {end: isPerWorker});
     worker.logFile = logStream;
   }
+
+  function setupExpresUsageForwarding(parentCtl) {
+    var debugExpress = require('./debug')('express-usage');
+    agent().on(
```

Comment:
if there is not parentctl channel, you don't have to do the agent().on(), I don't think, because there is no destination.